src/cookbook/effects/gradient-bubbles.md

```diff
@@ -65,26 +67,26 @@ background with a new stateless widget called
 represent the full-screen gradient that should be
 applied to the bubble.
 
-<!--skip-->
+<?code-excerpt "lib/main.dart (BubbleBackground)" replace="/return //g"?>
 ```dart
 BubbleBackground(
- // The colors of the gradient, which are different
- // depending on which user sent this message.
- colors: message.isMine
-   ? [const Color(0xFF6C7689), const Color(0xFF3A364B)]
-   : [const Color(0xFF19B7FF), const Color(0xFF491CCB)],
- // The content within the bubble.
- child: DefaultTextStyle.merge(
-   style: const TextStyle(
-     fontSize: 18.0,
-     color: Colors.white,
-   ),
-   child: Padding(
-     padding: const EdgeInsets.all(12.0),
-     child: child,
-   ),
- ),
-),
+  // The colors of the gradient, which are different
+  // depending on which user sent this message.
+  colors: message.isMine
+      ? [const Color(0xFF6C7689), const Color(0xFF3A364B)]
+      : [const Color(0xFF19B7FF), const Color(0xFF491CCB)],
```

Comment:
Sorry I forgot to make a suggestion for this one. You'll need to update here too:

```suggestion
      ? const [Color(0xFF6C7689), Color(0xFF3A364B)]
      : const [Color(0xFF19B7FF), Color(0xFF491CCB)],
```

Reply:
No worries! I just refreshed the code excerpt for it!